modules/cloud-function-v1/variables.tf

```diff
@@ -154,22 +154,35 @@ variable "trigger_config" {
 }
 
 variable "vpc_connector" {
-  description = "VPC connector configuration. Set create to 'true' if a new connector needs to be created."
+  description = "Populate this to create a VPC connector."
```

Comment:
description is not accurate, what if I populate with `create = false`, is it still creating?